Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM node:4-wheezy
```

Comment:
Nous ne prendrons pas le Dockerfile dans `master` parce que nous n'en avons pas besoin pour le moment et parce qu'il est de toute façon incomplet (chargement des données, dépendances manquantes).

Si tu veux faire ça bien, propose nous une PR dédiée et on itèrera.